try {
  age++;
} catch (error) {
  console.log(error.message);
}
